packages/firestore/lite/src/api/database.ts

```diff
@@ -0,0 +1,104 @@
+/**
+ * @license
+ * Copyright 2017 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as firestore from '../../';
+
+import { _getProvider } from '@firebase/app-exp';
+import { Provider } from '@firebase/component';
+import { FirebaseApp } from '@firebase/app-types';
+import { FirebaseService } from '@firebase/app-types/private';
```

Comment:
Should depend on `@firebase/app-types-exp` instead. 
It currently doesn't define a `FirebaseService` interface, I'm adding it in https://github.com/firebase/firebase-js-sdk/pull/3112/files